scipy-1.0/scipy-optimize.tex

```diff
@@ -21,66 +21,8 @@ \subsubsection*{Linear Optimization}
 
 \subsubsection*{Nonlinear Optimization}
 \paragraph{Local Minimization}
-
-The \texttt{minimize} function provides methods for finding local minima of
-nonlinear optimization problems. It unifies several methods with a common 
-interface in order to facilitate interchanging between solvers with different
-characteristics. Characteristics of the different methods are summarized in 
-Table~\ref{tab:minimize-si}.
-%% Currently, this introduces a ton of whitespace around a very simple expression (min f(x)). The reader can easily find the standard of meaning of `nonlinear programming', so I believe it should be removed. If we decide that it is appropriate to include, then we should add the entire problem statement - including constraints - and we should also include the entire linear programming problem statement in the previous section.
-% summarizes all the methods available for solving minimization problems of the type:
-%\begin{equation}
-%  \label{eq:minimization-prob}
-%  \min_x f(x)
-%\end{equation}
-%where $f$ is a multivariate function $f: \mathbb{R}^n \rightarrow \mathbb{R}$ and $x \in \mathbb{R}^n$.
-%%  Without the problem statement this sentence isn't really appropriate.
-%\texttt{COBYLA} and \texttt{SLSQP} methods support nonlinear constraints, \texttt{L-BFGS-B} and \texttt{TNC} allow box constraints $x^l \le x \le x^u$,
-%and the remaining methods are for unconstrained problems.
-
-% The available methods implement a variety of solution strategies.
-Derivative-free methods (e.g. \texttt{Nelder-Mead}, \texttt{Powell} and \texttt{COBYLA}) are appropriate for minimizing non-differentiable or
```

Comment:
It's a shame to lose lines 42 to 49. But if there's no space...

Reply:
Those derivative free solvers are old, right? If not <3 years old, it's out of scope.
It's good to concisely give context for the recent stuff, and that's why I kept the bit explaining the trust region approach. But all of the new stuff is trust region and derivative-based, so I don't think we can describe all the other types of optimization approaches.